iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesSensorTest.java

```diff
@@ -397,13 +396,6 @@ private void assertSecondaryLocation(Issue.Flow flow, int startLine, int startLi
     assertThat(issueLocation.textRange()).hasRange(startLine, startLineOffset, endLine, endLineOffset);
   }
 
-  private HelmProcessor mockHelmProcessor(Map<String, String> inputToOutput) {
```

Comment:
Was this not used anymore?

Reply:
Yes, it was only used by the deleted test